### DISCUSSION TITLE
Urgent: Restore deleted codespace

### DISCUSSION THREAD

**Discussion by bachts** [community/community#142307] [Codespaces]
### Select Topic Area

General

### Body

I accidentally deleted a codespace with uncommitted changes, but I need the files in there urgently for an upcoming deadline. I already opened a ticket for this #3061638, but it would be really appreciated if it could be resolved within a day.

**Comment by LakinduVirajith** [comment]
It’s tough when something like this happens, especially with a deadline approaching! Since you’ve already opened a ticket (#3061638), I’d recommend keeping an eye on your email for updates from GitHub Support. Unfortunately, once a Codespace is deleted, recovering uncommitted changes can be difficult, but the support team is your best bet for resolving this quickly.

I hope they can assist you soon!
